libvast_test/src/events.cpp

```diff
@@ -293,7 +305,22 @@ events::events() {
   SANITY_CHECK(zeek_dns_log, zeek_dns_log_slices);
   SANITY_CHECK(zeek_http_log, zeek_http_log_slices);
   SANITY_CHECK(bgpdump_txt, bgpdump_txt_slices);
-  //SANITY_CHECK(random, const_random_slices);
+  // SANITY_CHECK(random, const_random_slices);
+  // Read the full Zeek conn.log.
+  // TODO: port remaining slices to new deserialization API and replace
+  //       this hard-coded starting offset
+  id offset = 100000;
```

Comment:
This one could move down closer to its usage, e.g., right before the `for` loop.